modules/stochastic_tools/test/tests/reporters/statistics/statistics_timing.py

```diff
@@ -0,0 +1,58 @@
+import os
```

Comment:
Do we need to version-track this? If yes, maybe it would be better to have it in the python folder in STM.